add angular velocity and angular acceleration FFTs

dagar opened the pull request:
### DataPlotFFT
  - add angular velocity and acceleration FFTs
  - plot IMU_GYRO_NF_FREQ if set
  - use timestamp_sample if available
  - remove hard coded y range and amplitude scaling

![Screenshot from 2021-03-18 12-59-49](https://user-images.githubusercontent.com/84712/111666172-da07b500-87e9-11eb-9073-c615a549b909.png)

### DataPlotSpec
  - use timestamp_sample if available
  - allow zooming in y

![Screenshot from 2021-03-18 13-00-08](https://user-images.githubusercontent.com/84712/111666237-e5f37700-87e9-11eb-9bd2-ce071ed94a26.png)

bkueng commented on plot_app/configured_plots.py:
```diff
@@ -443,6 +443,7 @@ def generate_plots(ulog, px4_ulog, db_data, vehicle_data, link_to_3d_page,
     data_plot.add_graph(['control[0]', 'control[1]', 'control[2]'],
                         colors3, ['Roll', 'Pitch', 'Yaw'])
     if not data_plot.had_error:
+        data_plot._p.y_range = Range1d(0, 0.01)
```
Can you set `y_range=Range1d(0, 0.01)` in the constructor?

dagar replied:
Done.

dagar commented:
Rebased on master and comment addressed.

![Screenshot from 2021-05-31 13-57-58](https://user-images.githubusercontent.com/84712/120227996-435b6980-c218-11eb-9cba-6787b4c6f26c.png)

I also removed the explicit range for the PSD color scaling as it wasn't great in some circumstances.

#### Master
![Screenshot from 2021-05-31 13-59-58](https://user-images.githubusercontent.com/84712/120228138-874e6e80-c218-11eb-8bfb-a7b3e2e94f6d.png)

#### PR
![Screenshot from 2021-05-31 14-00-36](https://user-images.githubusercontent.com/84712/120228172-99301180-c218-11eb-9d7e-615914e20bbf.png)

bkueng commented on app/plot_app/plotting.py:
```diff
@@ -847,9 +853,9 @@ def add_graph(self, field_names, legends, window='hann', window_length=256, nove
+            time = time * 1.0e6 + self._cur_dataset.data[timestamp_key][0]
 
-            color_mapper = LinearColorMapper(palette=viridis(256), low=-80, high=0)
+            color_mapper = LinearColorMapper(palette=viridis(256))
```
The plot you show now misses the labels on the bar. Is there a way to get that back?
I want to make sure we can still compare. Otherwise you could set an explicit limit per plot.

bkueng reviewed (commented):
Did you gain any insights already?

dagar commented:
Updated.

<img width="856" alt="Screen Shot 2021-06-01 at 5 39 26 PM" src="https://user-images.githubusercontent.com/84712/120393686-577a9600-c300-11eb-803f-ec162cec2b6a.png">

dagar commented:
> Did you gain any insights already?

So far mainly a few airframe specifics, but maybe more generally seeing how valuable even a single well placed notch can be in many cases rather than the conservative low pass cutoff.
